### PR TITLE
Fix: Revise fragments to pass validation.

### PR DESCRIPTION
**Pull request by bjones1**

### CODE REVIEW

File: packages/fragment/tests/fragments/dl-2.xml [hunk 1]
```diff
@@ -1,11 +1,20 @@
 <FRAGMENT parents="p">
     <dl width="narrow">
-        <li><title>The Title</title>Item 1</li>
```
Comment (siefkenj) [inline packages/fragment/tests/fragments/dl-2.xml:3]: These ones I actually put in on purpose. I believe stuff like this is in the sample article and the contents of list items automagically gets wrapped in `<p>` tags.

File: packages/fragment/tests/fragments/special-symbols-1.xml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <FRAGMENT parents="p">
     <copyright />
     <phonomark/>
-    <copyleft />
```
Comment (siefkenj) [inline packages/fragment/tests/fragments/special-symbols-1.xml:4]: This one isn't in the schema but is in the sample article...Maybe we should file an issue to Rob to add it to the schema?

File: packages/fragment/tests/fragments/dl-2.xml (hunk 1 above)
Comment (bjones1) [reply packages/fragment/tests/fragments/dl-2.xml:3]: OK, I'll revert this and perhaps add a `readme.md` to explain why we include non-validating fragment. However, it seems weird to officially support broken PreTeXt...I wonder if it makes more sense to ask Rob to fix this in the sample article?

File: packages/fragment/tests/fragments/special-symbols-1.xml (hunk 2 above)
Comment (bjones1) [reply packages/fragment/tests/fragments/special-symbols-1.xml:4]: See https://github.com/PreTeXtBook/pretext/pull/2020.

File: packages/fragment/tests/fragments/dl-2.xml (hunk 1 above)
Comment (bjones1) [reply packages/fragment/tests/fragments/dl-2.xml]: Done!

**Pull request merged**